pkg/kubelet/server/stats/types.go

```diff
@@ -45,6 +45,8 @@ type NodeStats struct {
 	Memory *MemoryStats `json:"memory,omitempty"`
 	// Stats pertaining to network resources.
 	Network *NetworkStats `json:"network,omitempty"`
+	// Stats pertaining to total usage of filesystem resources.
+	Fs []FsStats `json:"fs,omitempty"`
```

Comment:
Done as you suggested, but still feel a bit nervous about having to support multiple devices down the road.  Maybe I should call this ContainerRootFs?